PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Please ensure your pull request adheres to the following guidelines:
+
+- [ ] Link additions should be added to the bottom of the relevant category.
+- [ ] New categories or improvements to the existing categorization are welcome.
+- [ ] Search previous pulls before making a new one, as yours may be a duplicate.
```

Comment:
You know, I would probably choose other text for it, to obscure for my taste :))

Reply:
K.I.S.S. ?